core/src/test/java/kafka/test/api/ShareConsumerTest.java

```diff
@@ -1832,83 +1835,129 @@ public void testShareAutoOffsetResetByDurationInvalidFormat() throws Exception {
             @ClusterConfigProperty(key = "unstable.api.versions.enable", value = "true")
         }
     )
+    @Timeout(90)
     public void testShareConsumerAfterCoordinatorMovement() throws Exception {
         setup();
         String topicName = "multipart";
         String groupId = "multipartGrp";
         Uuid topicId = createTopic(topicName, 3, 3);
         alterShareAutoOffsetReset(groupId, "earliest");
+        ScheduledExecutorService service = Executors.newScheduledThreadPool(5);
 
         try (Admin admin = createAdminClient()) {
```

Comment:
I think the admin client is only needed in the runnable that moves the coordinator.